src/main/resources/application.yml

```diff
@@ -67,6 +67,8 @@ logging:
 cloud:
   aws:
     enabled: true
+    mino-public-host: "http://minio.sandbox.fynarfin.io"
+    s3BaseUrl: "https://s3.ap-south-1.amazonaws.com"
     credentials:
```

Comment:
for credentials, make sure that it is using the same secret as bulk processor

Reply:
yes, it is using bulk secret